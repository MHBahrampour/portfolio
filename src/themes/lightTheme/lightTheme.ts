import { type ThemeOptions } from "@mui/material";

export const lightTheme: ThemeOptions = {
  palette: {
    mode: "light",
    common: {
      black: "#0f1b2c",
      white: "#f2f2f2",
    },
    primary: {
      main: "#0f1b2c",
      light: "#3f4856",
      dark: "#0a121e",
      contrastText: "#f2f2f2",
    },
  },
  shape: {
    borderRadius: 8,
  },
  typography: {
    fontFamily: "Poppins, sans-serif",
  },
};

// MUI Default Theme: https://mui.com/material-ui/customization/default-theme/
