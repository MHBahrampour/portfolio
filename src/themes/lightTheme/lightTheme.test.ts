import { lightTheme } from "./lightTheme";

it("checks lightTheme's snapshot", () => {
  expect(lightTheme).toMatchSnapshot();
});
