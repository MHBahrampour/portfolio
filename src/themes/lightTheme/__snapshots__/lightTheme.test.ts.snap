// Vitest Snapshot v1

exports[`checks lightTheme's snapshot 1`] = `
{
  "palette": {
    "common": {
      "black": "#0f1b2c",
      "white": "#f2f2f2",
    },
    "mode": "light",
    "primary": {
      "contrastText": "#f2f2f2",
      "dark": "#0a121e",
      "light": "#3f4856",
      "main": "#0f1b2c",
    },
  },
  "shape": {
    "borderRadius": 8,
  },
  "typography": {
    "fontFamily": "Poppins, sans-serif",
  },
}
`;
